k-distribution/include/kframework/builtin/kast.md

```diff
@@ -271,7 +271,7 @@ tail of the cell, and cells whose contents are sort `List` can only have `...`
 appear on at most one side in a single rule.
 
 ```k
-module RULE-CELLS
+module RULE-CELLS [implicit-module]
```

Comment:
The `docs/user_manual.md` needs to be updated (towards the end) with this attribute. There is a table there which says where which attributes are allowed.